mmv1/products/documentaiwarehouse/DocumentSchema.yaml

```diff
@@ -0,0 +1,262 @@
+# Copyright 2023 Google Inc.
```

Comment:
Is this resource file ready for review? Or are you still working on it?

Reply:
I just pushed a few final additions, it is ready now